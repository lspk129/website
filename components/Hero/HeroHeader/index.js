import styled from 'styled-components';

const HeaderContainer = styled.div`
  grid-row: 1 / 2;
  align-text: center;
  justify-self: center;
`;

export default HeaderContainer;
