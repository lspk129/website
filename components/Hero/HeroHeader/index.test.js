import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import HeroHeader from './';

describe('HeroHeader', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HeroHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div element', () => {
    const wrapper = shallow(<HeroHeader />);
    expect(wrapper).toBePresent();
    expect(wrapper).toHaveTagName('div');
  });
});
