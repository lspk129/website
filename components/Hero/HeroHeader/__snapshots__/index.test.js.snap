// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroHeader renders correctly 1`] = `
<div
  className="sc-bdVaJa ebKSmz"
/>
`;
